code/modules/mechs/components/arms.dm

```diff
@@ -44,3 +44,45 @@
 		to_chat(user, SPAN_NOTICE(" Actuator Integrity: <b>[round((((motivator.max_dam - motivator.total_dam) / motivator.max_dam)) * 100)]%</b>"))
 	else
 		to_chat(user, SPAN_WARNING(" Actuator Missing or Non-functional."))
+
+/obj/item/mech_component/manipulators/cheap
+	name = "lifter exosuit arms"
+	desc = "Industrial lifter arms that allow you to crudely manipulate things from the safety of your cockpit."
+	exosuit_desc_string = "industrial lifter arms"
+	icon_state = "loader_arms"
+	melee_damage = 15
+	max_damage = 90
+	power_use = 30
+
+/obj/item/mech_component/manipulators/light
+	name = "light arms"
+	exosuit_desc_string = "lightweight, segmented manipulators"
+	desc = "As flexible as they are fragile, these Vey-Med manipulators can follow a pilot's movements in close to real time."
+	icon_state = "light_arms"
+	melee_damage = 10
+	action_delay = 5
+	max_damage = 45
+	power_use = 10
+	matter = list(MATERIAL_STEEL = 10, MATERIAL_PLASTIC = 5)
+
+/obj/item/mech_component/manipulators/combat
+	name = "combat arms"
+	exosuit_desc_string = "flexible, advanced manipulators"
+	desc = "These advanced manipulators are designed for combat, and as a result can take and dish out beatings fairly well."
+	icon_state = "combat_arms"
+	melee_damage = 45 // Whack
+	action_delay = 10
+	max_damage = 125
+	power_use = 50
+	matter = list(MATERIAL_STEEL = 15, MATERIAL_PLASTEEL = 5, MATERIAL_PLASMA = 4, MATERIAL_DIAMOND = 2)
+
+/obj/item/mech_component/manipulators/heavy
+	name = "heavy arms"
+	exosuit_desc_string = "super-heavy reinforced manipulators"
+	icon_state = "heavy_arms"
+	desc = "Designed for durability and dishing out beatings, this heavy set of manipulators can both take and dish out beatings."
```

Comment:
Uhh, how about "Designed for prolonged combat situations, this heavy set of manipulators can both take and dish out beatings."